templates/template_pdf.html

```diff
@@ -483,152 +482,216 @@ <h4 class="ex2" align="center">
-                                        }
-                                        document.write(divd)
-                                        var ex17 = "<h5 align= center><u><a name=firstP>Pair-distance distribution analysis</a></u>   <a class='help1' style='background-color:#228B22;color:#000000;text-decoration:none font-size=16px' href='https://pdb-dev-beta.wwpdb.org/validation_help.html#pofr'>?</a> </h5>";
-                                        document.write(ex17)
-                                        var ex18 ='<p style=margin-bottom:0.5cm;> </p>';
-
-                                        ex18 += "<p class=ex2 align='justify'><u> P(r) analysis:</u> P(r) represents the distribution of distances between all pairs of atoms within the particle weighted by the respective electron densities. P(r) is the Fourier transform of I(s) (and vice versa). R<sub>g</sub> can be estimated from integrating the P(r) function. Agreement between the P(r) and Guinier-determined R<sub>g</sub> (table below) is a good measure of the self-consistency of the SAS profile. R<sub>g</sub> is a measure for the overall size of a macromolecule; e.g. a protein with a smaller R<sub>g</sub> is more compact than a protein with a larger R<sub>g</sub>, provided both have the same molecular weight (MW). The point where P(r) is decaying to zero is called D<sub>max</sub> and represents the maximum size of the particle.</p>";
-                                        ex18 +='<p style=margin-bottom:0.5cm;> </p>';
-                                        document.write(ex18);
-                                        var input_list = {{pddf_info}};
+																	</script>
+                                  
+																	<p style=margin-bottom:1cm;>
+
+                                  <h5 align=center>
+																		<u><a name=firstF> Flexibility analysis</a> </u>   <a class='help1' style='background-color:#228B22;color:#000000;text-decoration:none font-size=16px' href='https://pdb-dev-beta.wwpdb.org/validation_help.html#flexibility'>?</a>  
+																	</h5>
+
+                                  {% for i in range ( sasdb_code_html|length ) %}     
+																		<p align=justify>
+																			<u> Flexibility analysis for {{ sasdb_code_html[i] }}</u>: In a Porod-Debye plot, a clear plateau is observed for globular (partial or fully folded) domains, whereas, fully unfolded domains are devoid of any discernable plateau. For details, refer to Figure 5 in <a href=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3103662/>Rambo and Tainer, 2011</a>. In a Kratky plot, a parabolic shape is observed for globular (partial or fully folded) domains and a hyperbolic shape is observed for fully unfolded domains. 
+																		</p>
+                                  
+																		<div class=row>
+																			<div class=col-lg-12>
+																				<div class=box text-center container-fluid no-padding align=center style=height:60vh>
```

Comment:
Shouldn't `box text-center container-fluid no-padding` be quoted?

Reply:
I believe they should. In fact, at the beginning of this branch, I tested the `template_pdf.html` with [svenkreiss/html5validator](https://github.com/svenkreiss/html5validator) and there were about 600 issues. I started with stray tags and decided to continue with other issues later. The priority is to clean the rest of the JS inserts, so it would be much easier to deal with CSS-related things later.